src/dopri5.rs

```diff
@@ -86,7 +88,13 @@ where
             coeffs: Dopri54::new(),
             controller: Controller::default(x, x_end),
             out_type: OutputType::Dense,
-            rcont: [SVector::zero(); 5],
+            rcont: [
```

Comment:
`rcont: [OVector::zeros_generic(rows, cols); 5]` would save a few lines.

Reply:
Because `OVector` does not implement `Copy` this fails to build. We could add a trait bound like rust suggest for `Copy` but then this impl wouldn't be valid for `DVector` which acts more like a Vec

Errors:
```
error[E0277]: the trait bound `Matrix<T, D, Const<1_usize>, <DefaultAllocator as nalgebra::allocator::Allocator<T, D>>::Buffer>: Copy` is not satisfied
  --> src/dop853.rs:92:20
   |
92 |             rcont: [OVector::zeros_generic(rows, cols); 8],
   |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `Copy` is not implemented for `Matrix<T, D, Const<1_usize>, <DefaultAllocator as nalgebra::allocator::Allocator<T, D>>::Buffer>`
   |
   = help: the following implementations were found:
             <Matrix<T, R, C, S> as Copy>
   = note: the `Copy` trait is required because the repeated element will be copied

error[E0277]: the trait bound `Matrix<T, D, Const<1_usize>, <DefaultAllocator as nalgebra::allocator::Allocator<T, D>>::Buffer>: Copy` is not satisfied
  --> src/dopri5.rs:91:20
   |
91 |             rcont: [OVector::zeros_generic(rows, cols); 5],
   |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `Copy` is not implemented for `Matrix<T, D, Const<1_usize>, <DefaultAllocator as nalgebra::allocator::Allocator<T, D>>::Buffer>`
   |
   = help: the following implementations were found:
             <Matrix<T, R, C, S> as Copy>
   = note: the `Copy` trait is required because the repeated element will be copied
```

Reply:
Good point, let's keep it that way then.